components/Card.vue

```diff
@@ -173,12 +184,8 @@ export default Vue.extend({
      * @returns {string} Either an empty string, or a string representing the production year(s) for the current item or the relevant episode number of the item.
      */
     cardSubtitle(): string {
-      if (this.episode) {
-        if (this.item.IndexNumber) {
-          return this.$t('episodeNumber', {
-            episodeNumber: this.item.IndexNumber
-          }).toString();
-        }
```

Comment:
Probably out of the scope for this, but the title and subtitle for cards (especially for the up next/continue watching) need to be fixed

Reply:
Improved in #304